### ISSUE TITLE
FBTEE retrieval issue

### ISSUE THREAD

**Issue by linguistcrg** [CentreForDigitalHumanities/edpop-explorer#38]
While performing a search of the word "almanac", instead of getting this output:
`Performing query: almanac`

I get the following output:
```
Performing query: SQLPreparedQuery(where_statement='WHERE full_book_title LIKE ?', arguments=['%almanac%'])
EXCEPTION of type 'AttributeError' occurred with message: 'int' object has no attribute 'data'
```

And I don't get any results.

**Comment by tijmenbaarda** [comment]
This was caused by a small bug and will be fixed as part of #44.

**Comment by tijmenbaarda** [comment]
Closed by #44
